src/components/userProfile/components/SideMenu.js

```diff
@@ -15,11 +16,18 @@ export default class SideMenu extends React.Component {
                 className={MENU}
                 theme={DARK}
                 mode={INLINE}
+                selectedKeys={[location.pathname]}
```

Comment:
is `pathname` a built in method?

Reply:
The real question here is, what is [`location`](https://developer.mozilla.org/en-US/docs/Web/API/Location)? [`location`](https://developer.mozilla.org/en-US/docs/Web/API/Location) is a global object and `pathname` is a key on it that points to the exact path that your page is on. Your browser uses this [`location`](https://developer.mozilla.org/en-US/docs/Web/API/Location) object to keep track of the different parts of a `URL`.

Reply:
Oh I see...